### ISSUE TITLE
Formatting in plain html version of lecture 5a

### ISSUE THREAD

**Issue by richardreeve** [IBAHCM/RPiR#165]
In Lecture 5a - https://ibahcm.github.io/RPiR/lectures_html/RPiR5a-reporting.html - the emphasis code blocks:
```R
### <b>
```
and
```R
### </b>
```
appear in the html, when they should just disappear.

**Comment by soniamitchell** [comment]
They don't disappear here either https://ibahcm.github.io/RPiR/lectures_html/RPiR2c-development.html

Why do you think they should? Is there any documentation I can reference? Presumably they're being interpreted as R comments..

**Comment by richardreeve** [comment]
Yes, the docs are here - https://bookdown.org/yihui/rmarkdown/ioslides-presentation.html#code-highlighting - the text within those pseudo-html blocks should just be in bold, but if they are just removed in the html that would be fine. Or maybe `### <b>` could just be replaced by `<b>` in the html, and so on.

**Comment by soniamitchell** [comment]
Yes, but this describes the ioslides presentation output, not html_document output?

**Comment by richardreeve** [comment]
Yes, I know, but what I'm saying is that this text is supposed to formatting, but the html_document output doesn't know, and it would be nice to fix that if there's something we could do...

**Comment by soniamitchell** [comment]
Ahh... got it 👍 That will require some fiddling around but I think it might be doable...

**Comment by soniamitchell** [comment]
This fix broke html slide generation on the website.
